src/Hit/Git.hs

```diff
@@ -86,13 +86,25 @@ runAmend :: IO ()
 runAmend = do
     "git" ["add", "."]
     "git" ["commit", "--amend", "--no-edit"]
-    runPush True
+    runPush True False
 
 -- | @hit push@ command.
-runPush :: Bool -> IO ()
-runPush isForce = getCurrentBranch >>= \branch ->
-    "git" $ ["push", "--set-upstream", "origin", branch]
-         ++ ["--force" | isForce]
+runPush :: Bool -> Bool -> IO ()
+runPush isForce assignOwner = do
+    branch <- getCurrentBranch
+    "git" $ ["push", "--set-upstream", "origin", branch] ++ ["--force" | isForce]
+    when assignOwner (attemptAssignment branch)
+    where attemptAssignment :: Text -> IO ()
```

Comment:
As per our style guide th `where` keyword should on the separate line with the 2 spaces, like this:
```haskell
    bla bla
  where
    bla :: Bla -> Bla
    bla = ...
```

Reply:
This is no longer part of the source code / changes.